clang/test/Driver/rocm-detect-linux.hip

```diff
@@ -0,0 +1,9 @@
+// REQUIRES: system-linux
```

Comment:
This doesn't require linux, use an explicit triple

Reply:
agreed.  Will change to use target.